libs/gl-client/src/lnurl/tor_http_client.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::Result;
+use arti_client::{TorClient, TorClientConfig};
+use log::info;
+use tokio::io::{AsyncReadExt, AsyncWriteExt};
+use url::Url;
+
+pub struct TorHttpClient {
+    tor_client: TorClient<tor_rtcompat::PreferredRuntime>,
+}
+
+impl TorHttpClient {
+    pub async fn new() -> Result<Self> {
+        let mut config_builder = TorClientConfig::builder();
+        let address_filter = config_builder.address_filter();
+        address_filter.allow_onion_addrs(true);
+        address_filter.build()?;
+        config_builder.build()?;
```

Comment:
This call appears to be a duplicate of the one in the `create_bootstrapped` call, which uses `.unwrap()`. It'd be better to just pull this into the call below instead of doing a pre-flight check like this.